lib/matplotlib/figure.py

```diff
@@ -2046,6 +2046,23 @@ def dpi(self):
     def dpi(self, value):
         self._parent.dpi = value
```

Comment:
Should there be a `self.stale = True` here? (For symmetry with `set_dpi`?)

(Maybe not a question for you @scottjones03 though.)

Reply:
Maybe, but in practice the parent will catch this and trigger the required re-draw.